neo/SmartContract/StateReader.cs

```diff
@@ -104,6 +104,8 @@ public StateReader()
             Register("Neo.Transaction.GetOutputs", Transaction_GetOutputs);
             Register("Neo.Transaction.GetReferences", Transaction_GetReferences);
             Register("Neo.Transaction.GetUnspentCoins", Transaction_GetUnspentCoins);
+            Register("Neo.Transaction.GetUnspentCoins", Transaction_GetUnspentCoins);
+            Register("Neo.InvocationTransaction.GetScript", InvocationTransaction_GetScript);
```

Comment:
should be `Register("Neo.Transaction.GetScript", Transaction_GetScript)`

Reply:
hmm, only the subclass `InvocationTransaction` has the `Script` property. can we still just use `Transaction`?